src/apple_gpu_stats/README.md

```diff
@@ -8,8 +8,8 @@ Install xcode, make sure your command line is using the official Xcode app:
 xcode-select -switch ~/Application/Xcode.app
 ```
 
-You'll need to be a member of the W&B Apple Developer account, reachout to vanpelt@wandb.com.
+You'll need to be a member of the W&B Apple Developer account, reach out to vanpelt@wandb.com.
 
 Run `xcodebuild build` to build the binary, then copy the binary into vendor with:
 
-`cp build/Release/apple_gpu_stats ../wandb/bin/`
\ No newline at end of file
+`cp build/Release/apple_gpu_stats ../wandb/sdk/system/assets`
```

Comment:
Im not sure how i feel about this change... putting binaries into the source area feels strange to me.
And if you want to do this you need to change:
MANIFEST.in
otherwise this will break

Reply:
moved back